tests/source/fn-simple.rs

```diff
@@ -43,3 +44,5 @@ unsafe fn generic_call(cx: *mut JSContext, argc: libc::c_uint, vp: *mut JSVal,
 }
 
 pub fn start_export_thread<C: CryptoSchemee + 'static>(database: &Database, crypto_scheme: &C, block_size: usize, source_path: &Path) -> BonzoResult<mpsc::Consumer<'static, FileInstruction>> {}
+
+pub unsafe fn reborrow_mut(helloo: usize) -> Handle<NodeRef<marker::Mut, K, V, NodeType>, HandleType> { println!(yes); }
```

Comment:
Could you also add tests with a where clause and a return type that spans multiple lines?